packages/block-editor/src/components/link-control/style.scss

```diff
@@ -197,12 +197,14 @@ $preview-image-height: 140px;
 		// of text, particular those with no spaces.
 		// See: https://github.com/WordPress/gutenberg/issues/33586#issuecomment-888921188
 		white-space: pre-wrap;
+		overflow-wrap: break-word;
```

Comment:
Not related to this PR, but it seems that `word-wrap` has been renamed to [overflow-wrap](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap). 

>The property was originally a nonstandard and unprefixed Microsoft extension called word-wrap, and was implemented by most browsers with the same name. It has since been renamed to overflow-wrap, with word-wrap being an alias.

I guess we don't need both properties here and in other places we have this prop.

Reply:
Confirmed, looks like it's the same without it.

Reply:
I think we should remove the old one `word-wrap` and keep `overflow-wrap`, but it's a minor thing since it's an alias. Maybe we could replace all usages of `word-wrap` in another PR..

Reply:
I need to step away for a bit of a late lunch now. But feel free to push any changes to the PR. From my investigations, it looks like the key property that solved the problem was `word-break: break-all;`.